src/os/im/action/importaction.js

```diff
@@ -1,4 +1,5 @@
 goog.provide('os.im.action');
+goog.provide('os.im.action.ImportActionCallbackConfig');
```

Comment:
Moved into its own file. Figured it didn't need the externs treatment since it's OK to minify; but let me know putting it into os.externs is better for another reason.